### PR TITLE
create `@guardian/browserslist-config`

### PR DESCRIPTION
**Pull request by sndrs**
This adds a new `@guardian/browserslist-config` sharable config.

It's [generated from our google analytics data](https://github.com/browserslist/browserslist-ga-export), and provides support for browsers matching the following rule:

- support `<script type="module">` _and_ is used for **at least 0.01% of page views** on dotcom

_without giving too much  commercially sensitive info away 🕺, 0.01% would be 10s of thousands of page views for the last 30 days_

This means cutting the mustard on module support. At the current point in time, 99.76% of dotcom page views in browsers that caniuse tracks support modules via script tag.

image

it's hard to see what benefit bundling for `nomodule` users brings at this point.

### CODE REVIEW

File: packages/browserslist-config/README.md [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+
+```json
+  "browserslist": [
+    "extends @guardian/browserslist-config"
+  ]
+```
+
+## Rules
+
+-   support `<script type="module">` _and_ at least 0.01% usage
```
Comment (sndrs) [inline packages/browserslist-config/README.md]:
```suggestion
-   support `<script type="module">` _and_ at least 0.01% of page views on theguardian.com
```

**Comment by ashishpuliyel** [comment]
> it's hard to see what benefit bundling for nomodule users brings at this point.

Yes agree.

>  0.01% of page views on dotcom

This feels like an arbitrary number. How did you get to it?

**Comment by sndrs** [comment]
> This feels like an arbitrary number. How did you get to it?

it pretty much is. it seemed small enough to guarantee almost everyone will get working JS without getting too deep in to the weeds. very much open to a better number

**Comment by mchv** [comment]
Can we avoid relying on our `google analytics data` and rather use data from our own data warehouse? We only send data to `GA` when user does consent and we know that consent rate is very likely to reduce in the future.

**Comment by sndrs** [comment]
> Can we avoid relying on our `google analytics data` and rather use data from our own data warehouse? We only send data to `GA` when user does consent and we know that consent rate is very likely to reduce in the future.

my feeling is that this would represent the 20%. 

browserslist already provides tooling to get this data from GA, so in terms of this this PR, at least we know that we're at not just making a total guess (even if the data could be more accurate).

we would need to somehow create a way to get this data from our own warehouse in a way that browserslist can understand, and i have to confess i don't have the slightest idea who i'd even speak to about that, let alone how we could do it ourselves!

my inclination is to proceed with this and work on using the data warehouse as an improvement?

**Review by SiAdcock — APPROVED** [review]
🎉 🎉 🎉 🎉 🎉

**Pull request merged**